src/main/java/echo/Server.java

```diff
@@ -0,0 +1,91 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+
+
+public class Server {
+    public static void main(String[] args) throws IOException {
+        runEchoServer();
+    }
+
+    public static ServerSocket serverSocket = null;
+        public static Socket clientSocket = null;
+
+
+
+    public static void runEchoServer() throws IOException {
+
+        try {
+            serverSocket = createServerSocket(7777);
```

Comment:
Have you considered passing the port number to the server, to avoid hardcoding it here?

Reply:
Yes I can make that change.